src/lib/structures/Command.ts

```diff
@@ -84,6 +87,8 @@ const validationSchema = z
 			)
 			.array()
 			.nonempty(),
+		defaultMemberPermissions: z.any().optional(),
```

Comment:
Can this not be validated?

Reply:
i can't validate it because `Permission.FLAGS` is not enum